app/assets/stylesheets/admin/deploy_groups.scss

```diff
@@ -0,0 +1,3 @@
+.btn-disabled {
```

Comment:
all stylesheets are global, putting this here overrides the other definition ... this was removed to allow titles of disabled buttons to show ... why do you need the events to be none ?